### ISSUE TITLE
Training models on custom data sets, some questions！

### ISSUE THREAD

**Issue by YUANMU227** [longzw1997/Open-GroundingDino#72]
Thank you for your work, it's very good work.
I need to fine-tune the detection model on a custom dataset and have a few questions to confirm:
1. The training set needs to be in odvg data format. Can it only contain OD data and not VG data? In other words, the training set does not contain captions and phrases.
2. What to do if the custom category name is not included in the Label map of OD data?
Looking forward to your reply.
